### PR TITLE
auto OCP-21085

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei pls help to review. Thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.2/deployment.xyaml [hunk 1]
```diff
@@ -22,3 +22,13 @@ confirm_cancel_rollout:
   params:
     button_text: Yes, cancel
   action: click_button
+deploy_image_from_image_registry:
+  action: goto_deploy_image_page
+  action: search_image
+search_image:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.2/deployment.xyaml]: We can pass parameter to `set_input_value`

Comment (xiaocwan) [reply lib/rules/web/admin_console/4.2/deployment.xyaml]: Better to add a clear input action before input and search

**Comment by xiaocwan** [comment]
PR https://github.com/openshift/verification-tests/pull/354/files is pending on this one for reusing the rules.

**Comment by SSshahan** [comment]
@pruan-rht @akostadinov  please help to merge. Thanks

**Pull request merged**